### DISCUSSION TITLE
Midi panic icon

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1146] [Team Posts]
Just a quick poll ;-) : Which one of the icons looks better and should be kept? I am not sure myself...

![image](https://user-images.githubusercontent.com/10009541/73195105-9194c000-412d-11ea-8ee2-cd5673938727.png) ![image](https://user-images.githubusercontent.com/10009541/73195114-96597400-412d-11ea-909c-9744309f2044.png)

**Comment by terminator356** [comment]
Yeah, it did strike me as a bit odd, the proximity of the exclamation point to the drop down arrow
 gave the slight impression there were three buttons there.
I think the first icon probably says it better.
But I have a request: Change the colour so that it is not such a 'loud' primary red colour.
Maybe a bit lighter, more pastel-ish? Maybe not fill the whole square?
Primary red is too 'loud'. As in, "Hey man that suit you're wearing is loud!"  ;-)

**Comment by kybos** [comment]
Thanks. I did not like the second one either, that's why I was asking.
Too loud? It's midi panic after all, it's supposed to stick out ;-).
But you are possibly right that there is too much red. It works quite well on the darker themes, but on the default theme it looks a little strange...
I merged the first icon into master, just in case there is a new release soon. I will see what I can do then.

**Comment by terminator356** [comment]
Slight request: I think it is time to separate the style/fonts tab into two tabs - style and fonts.
The dialog is now too long for me and the bottom is cut off requiring alt/move just to see the Apply/OK buttons.
I could take care of it now but I don't want to step on anything you might be doing.
Should I do it or do you want to do it?

**Comment by kybos** [comment]
What is your screen resolution? The dialog is 702 (logical) pixels high (using 10pt font size), that's not so much, given a normal HD screen has 1080. I know laptop screens can be smaller, so it would be good to know what the max size of dialogs should be as a rule.
In the long term it certainly makes sense to separate the tabs anyway, there could be more options added etc. I can do it, I assume you have other things to do before the release.

**Comment by terminator356** [comment]
My screen is 1366x768 (old).
The dialog seems to be about 850 pixels high here on KDE.
I'd say 768 is usually a good maximum, at least for standard def.
It covers older standard LCD screens like this one.
But don't forget the taskbar, at least here, eats up about 40 pixels of that.

**Comment by kybos** [comment]
There is a PR for you to check ;-).
I put the style sheet path to a separate group box, also because I have more options in preparation related to the style sheets...

**Comment by spamatica** [comment]
Den tis 28 jan. 2020 kl 07:14 skrev Tim <notifications@github.com>:

> My screen is 1366x768 (old).
> The dialog seems to be about 850 pixels high here on KDE.
> I'd say 768 is usually a good maximum, at least for standard def.
> It covers older standard LCD screens like this one.
>

I agree it's good to cater for 1366x768 but, really you should try to get a
bigger monitor :D
Though I admire your tenacity with sticking to working hardware! .\../

**Comment by kybos** [comment]
Somewhat less intrusive and I think acceptable on all 3 themes:

![image](https://user-images.githubusercontent.com/10009541/73346137-a5ebd080-4285-11ea-818e-d9c300504d2c.png)  ![image](https://user-images.githubusercontent.com/10009541/73346189-bdc35480-4285-11ea-937a-5a11995025fd.png)

OK for you?

**Comment by terminator356** [comment]
Hey, that's neat! Different. I think it says a lot, being a midi function. Direct. To the point.

**Comment by spamatica** [comment]
I concurr, that's nice

Den ons 29 jan. 2020 kl 15:46 skrev Tim <notifications@github.com>:

**Comment by kybos** [comment]
Thanks guys.
I will push it tomorrow, it's on another PC...
